test/specs/mounting-options/provide.spec.js

```diff
@@ -86,8 +106,23 @@ describeWithShallowAndMount('options.provide', mountingMethod => {
     }
   )
 
+  itDoNotRunIf(
+    !injectSupported,
+    'injects in a composition api component',
+    () => {
+      const localVue = createLocalVue()
+      localVue.use(VueCompositionApi)
+      const wrapper = mountingMethod(CompositionComponentWithInject, {
+        provide: { fromMount: '_' },
+        localVue
+      })
+
+      expect(wrapper.html()).to.contain('_')
+    }
+  )
+
   it('config with function throws', () => {
-    config.provide = () => {}
+    config.provide = () => { }
```

Comment:
Ah. This is the guy breaking the linter. `npm run format` should clean that up for you in the future. My apologies for the runaround

Reply:
🤞